packages/terra-popup/examples/Index.site-page.jsx

```diff
@@ -19,83 +14,59 @@ import PopupNoHeader from './site-examples/PopupNoHeader';
-import PopupMiddleLeft from './site-examples/PopupMiddleLeft';
-import PopupMiddleRight from './site-examples/PopupMiddleRight';
-import PopupTopCenter from './site-examples/PopupTopCenter';
-import PopupTopLeft from './site-examples/PopupTopLeft';
-import PopupTopRight from './site-examples/PopupTopRight';
 import PopupInsideModal from './site-examples/PopupInsideModal';
 import PopupAttachmentBehavior from './site-examples/PopupAttachmentBehavior';
+import PopupAttachments from './site-examples/PopupAttachments';
 
-const PopupExamples = () => (
-  <div>
-    <div id="version">Version: {version}</div>
-    <Markdown id="readme" src={ReadMe} />
-    <IndexTemplate
-      title="Popup Default Props"
-      example={<PopupStandard />}
-      exampleSrc={PopupStandardSrc}
-    />
-    <h2 id="arrow-popup" style={{ paddingTop: '20px' }}>Popup Display With Arrow</h2>
-    <PopupWithArrow />
-    <h2 id="popup-attachment-behaviors" style={{ paddingTop: '20px' }}>Popup Attachment Behavior</h2>
-    <PopupAttachmentBehavior />
-    <h2 id="alignment-popups" style={{ paddingTop: '20px' }}>Popup Attachments</h2>
-    <Grid>
-      <Grid.Row>
-        <Grid.Column col={3}>
-          <PopupBottomLeft />
-        </Grid.Column>
-        <Grid.Column col={3}>
-          <PopupBottomCenter />
-        </Grid.Column>
-        <Grid.Column col={3}>
-          <PopupBottomRight />
-        </Grid.Column>
-      </Grid.Row>
-      <Grid.Row>
-        <Grid.Column col={3}>
-          <PopupMiddleLeft />
-        </Grid.Column>
-        <Grid.Column col={3}>
-          <PopupMiddleCenter />
-        </Grid.Column>
-        <Grid.Column col={3}>
-          <PopupMiddleRight />
-        </Grid.Column>
-      </Grid.Row>
-      <Grid.Row>
-        <Grid.Column col={3}>
-          <PopupTopLeft />
-        </Grid.Column>
-        <Grid.Column col={3}>
-          <PopupTopCenter />
-        </Grid.Column>
-        <Grid.Column col={3}>
-          <PopupTopRight />
-        </Grid.Column>
-      </Grid.Row>
-    </Grid>
-    <h2 id="dimension-popup" style={{ paddingTop: '20px' }}>Popup Size Variants</h2>
-    <PopupDimensions />
-    <h2 id="bounded-popup" style={{ paddingTop: '20px' }}>Popup Bounded</h2>
-    <PopupBounded />
-    <h2 id="no-header-popup" style={{ paddingTop: '20px' }}>Popup Bounded With No Header</h2>
-    <PopupNoHeader />
-    <h2 id="class-name-popup" style={{ paddingTop: '20px' }}>Popup Adding Class Styles</h2>
-    <PopupClassName />
-    <IndexTemplate
-      title="Popup Inside Modal"
-      example={<PopupInsideModal />}
-      exampleSrc={PopupInsideModalSrc}
-    />
-    <br />
-    <PropsTable id="props" src={PopupSrc} />
-  </div>
+const DocPage = () => (
+  <DocTemplate
+    packageName={name}
+    readme={ReadMe}
+    srcPath={`https://github.com/cerner/terra-framework/tree/master/packages/${name}`}
+    examples={[
+      {
+        title: 'Popup Default Props',
+        example: <PopupStandard />,
```

Comment:
Should these have a source? Or should the doc template conditionally render the example?

![image](https://user-images.githubusercontent.com/6720991/39833623-408ac074-5390-11e8-8213-720e5f8a3c52.png)

Reply:
Both. I wasn't intending to add src for each popup example- updating now but the template should also be conditional IMO